jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -43,7 +43,7 @@
 
     /** One slot cache */
     public static <Key, Value> Cache<Key, Value> createOneSlotCache() {
-        return new Cache1<>() ;
+        return new CacheGuava<>(1) ;
     }
```

Comment:
Shall I check and see whether `Cache1` is really in use? It tastes to me like memoization, which might be better supplied by some other means in Java 8, e.g via statics that use [Supplier](http://docs.oracle.com/javase/8/docs/api/java/util/function/Supplier.html).